src/api/authorData.js

```diff
@@ -38,9 +38,9 @@ const deleteSingleAuthor = (firebaseKey) => new Promise((resolve, reject) => {
 const updateAuthor = () => {};
 
 // TODO: GET A SINGLE AUTHOR'S BOOKS   use thissss!!!!!!!!!!!
-const getbooksByAuthor = (authorId) => new Promise((resolve, reject) => {
-  axios.get(`${dbUrl}/books.json?orderBy="author_id"&equalTo="${authorId}"`)
-    .then((response) => resolve(Object.values(response.data)))
+const getbooksByAuthor = (firebaseKey) => new Promise((resolve, reject) => {
+  axios.get(`${dbUrl}/books.json?orderBy="author_id"&equalTo="${firebaseKey}"`)
+    .then((response) => console.warn(Object.values(response.data)))
```

Comment:
```suggestion
    .then((response) => resolve(Object.values(response.data)))
```

Whoops! Forgot to change this back.